src/routes/settings/feedback.ts

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithoutClientOrSupplier } from '../../comms/controller';
+
+interface Request {
+  body: {
+    theme: string;
```

Comment:
**requested change**

I think you need to change it to this to match the Joi schema below:
```suggestion
    theme?: string;
```